src/stores/Entities/hooks.ts

```diff
@@ -1,131 +1,55 @@
 import { singleCallSettings } from 'context/SWR';
-import { useZustandStore } from 'context/Zustand/provider';
 import { useEffect } from 'react';
-import { GlobalStoreNames } from 'stores/names';
 import useSWR from 'swr';
-import { Schema } from 'types';
 import { ESTUARY_SUPPORT_ROLE } from 'utils/misc-utils';
 import { useShallow } from 'zustand/react/shallow';
-import { EntitiesState } from './types';
+import { useEntitiesStore } from './Store';
 
-export const useEntitiesStore_setCapabilities = () => {
-    return useZustandStore<EntitiesState, EntitiesState['setCapabilities']>(
-        GlobalStoreNames.ENTITIES,
-        (state) => state.setCapabilities
-    );
+export const useEntitiesStore_capabilities_readable = () => {
+    return useEntitiesStore((state) => ({
+        ...state.capabilities.admin,
+        ...state.capabilities.write,
+        ...state.capabilities.read,
+    }));
 };
 
-export const useEntitiesStore_capabilities = (
-    kind: keyof EntitiesState['capabilities']
-) => {
-    return useZustandStore<EntitiesState, Schema>(
-        GlobalStoreNames.ENTITIES,
-        (state) => state.capabilities[kind]
-    );
```

Comment:
We should go in this direction (where we pass `admin`,`read`,etc.) to fetch the capabilities we want - but the other ones are used right now so just removing